pkgs/tools/misc/cpu-energy-meter/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libcap
+}:
+stdenv.mkDerivation rec {
+  pname = "cpu-energy-meter";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "sosy-lab";
+    repo = "cpu-energy-meter";
+    rev = version;
+    hash = "sha256-QW65Z8mRYLHcyLeOtNAHjwPNWAUP214wqIYclK+whFw=";
+  };
+
+  patchPhase = ''
+    runHook prePatch
```

Comment:
patchPhase should always be left as is, use pre and postPatch instead

Reply:
Fixed in 2808b9721c4a